### PR TITLE
KAFKA-2447: Add capability to KafkaLog4jAppender to be able to use SSL

### PR DESCRIPTION
**Pull request by SinghAsDev**

### CODE REVIEW

**Comment by SinghAsDev** [comment]
@guozhangwang could you help reviewing this tiny patch.

**Comment by SinghAsDev** [comment]
The failed test is not related to the changes in this PR. There is already a JIRA to fix the test, https://issues.apache.org/jira/browse/KAFKA-2455.

**Comment by granders** [comment]
@guozhangwang do you mind giving this a quick review?

File: log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java [hunk 1]
```diff
@@ -42,10 +42,22 @@
     private static final String RETRIES_CONFIG = "retries";
     private static final String KEY_SERIALIZER_CLASS_CONFIG = "key.serializer";
     private static final String VALUE_SERIALIZER_CLASS_CONFIG = "value.serializer";
+    private static final String SECURITY_PROTOCOL = "security.protocol";
+    private static final String SSL_TRUSTSTORE_LOCATION = "ssl.truststore.location";
+    private static final String SSL_TRUSTSTORE_PASSWORD = "ssl.truststore.password";
+    private static final String SSL_KEYSTORE_TYPE = "ssl.keystore.type";
+    private static final String SSL_KEYSTORE_LOCATION = "ssl.keystore.location";
+    private static final String SSL_KEYSTORE_PASSWORD = "ssl.keystore.password";
```
Comment (ijuma) [inline log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java]: Why are we not using the constants in `SSLConfigs`?

**Comment by gwenshap** [comment]
LGTM

**Comment by ijuma** [comment]
This caused some compiler errors, fix: https://github.com/apache/kafka/pull/368

**Pull request closed without merging**